ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.12.yaml

```diff
@@ -0,0 +1,29 @@
+base_images:
+  upi-installer:
+    name: "4.12"
+    namespace: origin
+    tag: upi-installer
+releases:
+  latest:
+    candidate:
+      product: okd
+      stream: okd-scos
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-ovn
+  interval: 48h
```

Comment:
not required - release-controller would start the job whenever necessary (on a new nightly)

Reply:
Nevermind, its okay to have it